index.js

```diff
@@ -6,8 +6,8 @@ function isSlitherInstalled() {
   return shelljs.which("slither");
 }
 
-function executeSlither(astFile) {
-  shelljs.exec(`slither ${astFile}`);
+function executeSlither(astFile, flags = '') {
+  shelljs.exec(`slither ${astFile} ${flags}`);
```

Comment:
Might be nice to be able to have an array of flag-strings:

```suggestion
  shelljs.exec(`slither ${astFile} ${Array.isArray(flags) ? flags.join(' ') : flags}`);
```